doc/desktop/DevEnvSetup.md

```diff
@@ -1,37 +1,52 @@
-# Prerequisites:
-lein, node.js v.8 , cmake, Qt 5.9.1 (with QtWebEngine components installed), Qt's qmake available in PATH
+# Development environment setup
```

Comment:
I think this file could be simply deleted as it is already in docs.status.im

Reply:
Although the docs.status.im page doesn't cover the desktop build